sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/PluginManifest.java

```diff
@@ -320,6 +326,21 @@ public PluginManifest setUseChildFirstClassLoader(boolean useChildFirstClassLoad
   }
 
   /**
+   * @since 1.16
+   */
+  public boolean isSonarLintPlugin() {
+    return sonarlintPlugin;
+  }
+
+  /**
+   * @since 1.16
+   */
+  public PluginManifest setSonarLintPlugin(boolean sonarlintPlugin) {
```

Comment:
Same feedback. Method name is strange as it's a boolean. I suggest supportsSonarLint(boolean) and isSonarLintSupported()